index.html

```diff
@@ -7,7 +7,7 @@
     <link rel="stylesheet" href="styles/style.css">
   </head>
   <body>
-    <header>
+    <header class="top_header">
       <div class="header" >
```

Comment:
Having a class called `header` is a bit confusing, especially since it's inside the `<header>` section of the body.